components/bl-pdfViewer-component/src/index.js

```diff
@@ -49,15 +55,15 @@ export default function PdfViewer({ component, eventHandlers }) {
     }
 
     if (Number(target.value)) {
-      const ensureRange = (v, {min, max}) => Math.max(min, Math.min(v, max));
+      const ensureRange = (v, { min, max }) => Math.max(min, Math.min(v, max));
```

Comment:
Function without dependencies should be declared outside of another function/component